src/relay/ir/pretty_printer.cc

```diff
@@ -664,7 +664,9 @@ class PrettyPrinter :
   Doc PrintAttr(const NodeRef& value, bool meta = false) {
     if (value.defined()) {
       Doc printed_attr;
-      if (meta) {
+      if (value.as<tvm::ir::Any>()) {
+        printed_attr << "?";
```

Comment:
could this be ambiguous?  why not print "any"

Reply:
I kind of like `?` instead of `any` because the shape will still be checked dynamically, so even though we're calling the node "any," not literally any shape will work